### PR TITLE
CVE-2015-6778 and CVE-2011-1446

### PR DESCRIPTION
**Pull request by Bindernews**

### CODE REVIEW

File: cves/CVE-2011-1446.yml [hunk 1]
```diff
@@ -25,7 +25,9 @@ reviews:
 - 6865026
 - 6826015
```
Comment (andymeneely) [inline cves/CVE-2011-1446.yml]: I need the description and CWE too

File: cves/CVE-2011-1446.yml [hunk 2]
```diff
@@ -54,8 +56,8 @@ unit_tested:
 
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
-  answer: 
-  code: 
+  answer:
```
Comment (andymeneely) [inline cves/CVE-2011-1446.yml]: I need written answers too to justify it. If the tests exist, then find them and say where they are. And, I need an answer to "fix:" as well.

File: cves/CVE-2011-1446.yml [hunk 3]
```diff
@@ -72,11 +74,12 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer: 
+  answer: |
+    The bug was reported three times by the same reporter.
```
Comment (andymeneely) [inline cves/CVE-2011-1446.yml]: I need more information than that. This sounds like it has a story behind it.

File: cves/CVE-2011-1446.yml [hunk 4]
```diff
@@ -72,11 +74,12 @@ discovered:
...
   date: 
   automated: 
-  google: 
-  contest: 
+  google: false
+  contest: false
 subsystem:
```
Comment (andymeneely) [inline cves/CVE-2011-1446.yml]: This one doesn't look done - I need this completed ASAP

Comment (Primalink) [inline cves/CVE-2011-1446.yml]: Like Professor Meneely said, this CVE does not look like it is finished. The majority of the fields need to be filled out and you could elaborate on your "discovered answer" field.

File: cves/CVE-2015-6778.yml [hunk 5]
```diff
@@ -74,20 +87,25 @@ discovered:
...
 
     Look at the path of the source code files code that were fixed to get
     directory names. Look at comments in the code. Look at the bug reports how
     the bug report was tagged.
-  answer: 
-  name: 
+  answer: |
+    The vulenrability was in the pdfium library which is the PDF library Chromium uses, specially in
+    the JBIG2 image handling portion of the pdfium library. In the bug report it's listed as
+    "Internals > Plugins > PDF".
```
Comment (Primalink) [inline cves/CVE-2015-6778.yml]: This is a good answer, but the first sentence feels kinda long. Making the sentence shorter or splitting it into two sentences could make it more readable.

File: cves/CVE-2015-6778.yml [hunk 6]
```diff
@@ -96,10 +114,14 @@ interesting_commits:
...
     emerging themes?
   commits:
-  - commit: 
-    note: 
-  - commit: 
-    note: 
+  - commit: 06f255ca4040f3d73cf09f1dbd63a3b167f6a4c1
+    note: |
+      It seems commit 8a9ce577 was contentious as this commit fixes one malloc/free bug created in that commit. 
+      It appears the rewviewers weren't as thorough as they should have been.
```
Comment (cgv2921) [inline cves/CVE-2015-6778.yml]: "rewviewers" should be reviewers

**Pull request merged**